frontend/packages/dev-console/src/extensions/add-actions.ts

```diff
@@ -0,0 +1,29 @@
+import { Extension } from '@console/plugin-sdk/src/typings/base';
+import { AccessReviewResourceAttributes } from '@console/internal/module/k8s';
+
+namespace ExtensionProperties {
+  export type AddAction = {
+    /** Optional ID used to identify the action. */
+    id?: string;
```

Comment:
I'd suggest making `id` mandatory for both safety and consistency (assuming there can be multiple extension instances with the same `label`).

Reply:
Also, we can add `data-test-id={id}` to `CatalogTile` component to preserve existing behavior.